firecares/utils/arcgis2geojson.py

```diff
@@ -0,0 +1,229 @@
+"""
```

Comment:
NICE!  In the past, my method for converting ArcGIS REST json to geojson was running an `ogr2ogr` task in the background and reading the results from the resulting file (blech), this is SO MUCH BETTER!